diff --git a/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/compile.js b/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/compile.js
index 965b24b..f13c809 100644
--- a/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/compile.js
+++ b/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/compile.js
@@ -35,9 +35,6 @@ async function getConfigFile(options) {
     }
     let path = await (0, find_up_1.default)(options.config);
     if (path && path.length) {
-        if (process.platform === "win32") {
-            path = `file:///${path}`;
-        }
         const configModule = await Promise.resolve().then(() => __importStar(require(path)));
         return configModule.default || configModule;
     }
